addons/account_peppol/tests/test_peppol_messages.py

```diff
@@ -244,16 +250,15 @@ def test_send_success_message(self):
             checkbox_send_peppol=True,
         )
 
-        with self._patch_peppol_requests():
-            wizard.action_send_and_print()
+        wizard.action_send_and_print()
 
-            self.env['account_edi_proxy_client.user']._cron_peppol_get_message_status()
-            self.assertRecordValues(
-                move, [{
-                    'peppol_move_state': 'done',
-                    'peppol_message_uuid': FAKE_UUID[0],
-                }])
-            self.assertTrue(bool(move.ubl_cii_xml_id))
+        self.env['account_edi_proxy_client.user']._cron_peppol_get_message_status()
+        self.assertRecordValues(
+            move, [{
+                'peppol_move_state': 'done',
+                'peppol_message_uuid': FAKE_UUID[0],
+            }])
```

Comment:
(cosmetic comment)
```suggestion
        self.assertRecordValues(move, [{
                'peppol_move_state': 'done',
                'peppol_message_uuid': FAKE_UUID[0],
            }],
        )
```
or
```suggestion
        self.assertRecordValues(
            move,
            [{
                'peppol_move_state': 'done',
                'peppol_message_uuid': FAKE_UUID[0],
            }],
        )
```
or
```suggestion
        self.assertRecordValues(
            move,
            [{'peppol_move_state': 'done', 'peppol_message_uuid': FAKE_UUID[0]}],
        )
```

(I don't like the unclosed indentation due to the missing closing parenthesis not aligned with the opening one)